src/Transport/Http/HttpStatusCode.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * This file is part of the prooph/event-store-client.
+ * (c) 2018-2018 prooph software GmbH <contact@prooph.de>
+ * (c) 2018-2018 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\EventStoreClient\Transport\Http;
+
+class HttpStatusCode
```

Comment:
Why not make this class instantiable (with a defined code), that way you can typehint and pass around known good statuses (and even include some grouping logic (e.g. error/redirection/whatever).

Reply:
You mean making it an enum? Not sure if that's really needed, I will check again.